bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/PowerControlPower.java

```diff
@@ -29,20 +32,44 @@
 @NonNullByDefault
 public class PowerControlPower extends BaseChannelHandler<CommandConfirmation> {
     private final Logger logger = LoggerFactory.getLogger(PowerControlPower.class);
+    private final ConfigProvider configProvider;
+    private ScheduledExecutorService scheduler;
+    private static final int WOL_PACKET_RETRY_COUNT = 10;
```

Comment:
Isn't it too much to send the packet 10 times ?

Reply:
this is based on the implementation in SamsungTvHandler  and reused the wakeonlanutility class.